fastlane/Fastfile

```diff
@@ -55,21 +55,23 @@ platform :ios do
     crashlytics(
       crashlytics_path: './Pods/Crashlytics/',
       groups: options[:group],
-      notifications: 'true'
+      notifications: 'true',
+      notes: options[:notes]
```

Comment:
@jshier What about having them default to an empty string? Or would you prefer we not include the notes parameter if there are no notes being passed in?